api/app/tests/apps/users/factories.py

```diff
@@ -8,7 +8,7 @@ class Meta:
         model = settings.AUTH_USER_MODEL
 
 
-class SuperUserFacotry(UserFactory):
+class SuperUserFactory(UserFactory):
```

Comment:
Deze subclass zet geen staff of superuser eigenschappen op de gecreëerde user? Wat voegt ie toe?